src/books.py

```diff
@@ -0,0 +1,35 @@
+from flask import Blueprint, jsonify, request
+from flasgger import swag_from
+
+from src.models import Book
+from src.constants.http_status_codes import HTTP_200_OK, HTTP_404_NOT_FOUND
+
+
+books_bp = Blueprint('books', __name__, url_prefix='/api')
+
+
+@books_bp.get("/books/available")
+@swag_from("./docs/books/available_books.yaml")
+def available_books():
+
+
+    books = Book.query.filter_by(is_available=True)
+
+    if len(list(books)) < 1:
+        
+        return jsonify({
+            "error": "no books are available"
+        }),HTTP_404_NOT_FOUND
+
+    books_object= list()
```

Comment:
It is more pythonic to use the literal `[]` since you just need an empty list anyway. See this discussion on [stackoverflow](https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better)